data_model.md

```diff
@@ -0,0 +1,449 @@
+file with multiple observations.
+
+A framework user uploads a Forecast using the same process as for
+uploading Observations, except that a Forecast is described by a
+different set of attributes described later. Each Variable in the
+forecast (e.g., GHI, air temperature, wind speed) is uploaded as a
+separate time series comprising Time, Value pairs.
+
+Metadata
+--------
+{: .anchor}
+
+This section outlines the primary metadata containers in the Solar
+Forecast Arbiter: **Sites**, **Observations** and **Forecasts**, and
+**Aggregates**. All time series data is linked to an observation or a
+forecast, which is linked to a site or an aggregate. The figure below
+outlines the relationships among sites, forecasts and observations, and
+their associated time series data.
+
+<img src="/images/datamodel.png" alt="Figure of site, observation and forecast relationships" class="figure">
+<figcaption class="figure">Site with associated Forecasts (green) and Observations (blue). Each
+variable is associated with a distinct Forecast and Observation.</figcaption>
+<br/>
+A **Site** describes a physical location that will have Observations and
+Forecasts associated with it. For example, a solar power plant is
+entered into Solar Forecast Arbiter as a Site which has many
+Observations and Forecasts associated with it (e.g. temperature, GHI, AC
+power). In turn, Observations and Forecasts rely on the associated Site
+for location information.
+
+An **Observation** or **Forecast** contain timeseries data and specify
+the data’s associated variable, units, and Site. Forecasts and
+Observations require specification of the variable being described in
+the provided data so that they can be properly paired for analysis (see
+[*Variables and Units*](#variables-and-units)). Forecasts and Observations
+each have additional required parameters described below.
+
+An **Aggregate** is an aggregation of many Observations. The Sites
+associated with each Observation identify the geographic locations a
+forecast provider may use to create a single aggregated Forecast. The
+framework may generate an aggregate Observation from the listed
+Observations.
+
+An **Organization** groups Sites, Observations, Forecasts and Users.
+Organizations provide a logical partition on which to sort objects and
+create access control rules. Organizations are not otherwise essential
+to the data model and are not described in depth here.
+
+Creation of new objects will require valid metadata for all required
+parameters. Most users will use the Solar Forecast Arbiter dashboard to
+create new objects. The API will also support creation via uploading a
+valid JSON definition of the object. Metadata for sites, forecasts, and
+observations will be returned as JSON in the Solar Forecast Arbiter API.
+
+### Site
+{: .anchor}
+
+
+Each Site will have a number of required parameters that identify the
+Site. They are:
+
+-   Name: name for the Site, e.g. Desert Rock
```

Comment:
It appears that Name is used in file parsing as a keyword.  Can we distinguish keywords with different font?  E.g., `name` (not capitalized).

Reply:
I'm ok with doing something like italicizing the key terms, but we want to rely on the API documentation for detail on format of what people should upload e.g. https://dev-api.solarforecastarbiter.org/#tag/Sites/paths/~1sites~1/post 

Keep in mind the "dashboard" form interface will automatically do this for most people.

Reply:
OK. I'm sure I've raised this before, it's a reflex reaction because we're writing this description without having the accompanying API documents as context.

Reply:
Hm, as far as we're concerned, https://dev-api.solarforecastarbiter.org/ is the detailed API documentation. Or at least the prototype of it. (It does need minor updates, though.)

I wanted to write this document to give context to the API, though I don't know if that was ever communicated well to the team.